webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -177,6 +200,13 @@ export default {
       users: [],
       contentMin: 3,
       hashtags: [],
+      elem: null,
+      blurImage: false,
+    }
+  },
+  mounted() {
+    if (this.contribution && this.contribution.blurImage === true) {
+      this.blurImage = true
```

Comment:
@ogerly please check how all the other form values are initiated – I know it's a bit confusing but I think we should stick to the existing behavior.

This would also mean we don't need the `mounted()` block anymore, which is always good, and we don't need to double the value for blurring images as `checkbox` and `blurImage` – it could just be one `blurImage` value that responds to the checkbox input!

Reply:
mounted and put everything on blurImage. 
i hope i understood that correctly :)